.github/workflows/ci.yml

```
@@ -43,30 +43,39 @@ jobs:
-  benchmarks:
-    name: Benchmarks
+  perf:
+    name: Perf goldens
     runs-on: ubuntu-24.04
-    permissions:
-      contents: read # required for actions/checkout
-      id-token: write # required for OIDC authentication with CodSpeed
     steps:
       - uses: actions/checkout@v4
       - uses: pnpm/action-setup@v2
       - uses: actions/setup-node@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)